jest.config.js

```diff
@@ -6,11 +6,7 @@ const { pathsToModuleNameMapper } = require('ts-jest/utils');
 const { compilerOptions } = require('./tsconfig.base');
 
 module.exports = {
-    setupFilesAfterEnv: [
-        'jest-dom/extend-expect',
-        'react-testing-library/cleanup-after-each',
```

Comment:
react-testing-library/cleanup-after-each is now deprecated and does not do anything. let me see if i can find the original msg

Reply:
since 9.0.0:

```
cleanup-after-each: If any of your code has import '@testing-library/react/cleanup-after-each' then you need to remove that. It's happening automatically now.
```

Reply:
deprecation message:
```

  'The module `@testing-library/react/cleanup-after-each` has been deprecated and no longer does anything (it is not needed). You no longer need to import this module and can safely remove any import or configuration which imports this module',



```

Reply:
Ah ok cool 👍